charts/prometheus/templates/prometheus-alerts-configmap.yaml

```diff
@@ -101,11 +103,24 @@ data:
             tier: airflow
             component: container
             workspace: {{ printf "%q" "{{ $labels.workspace }}" }}
+            deployment: {{ printf "%q" "{{ $labels.deployment }}" }}
```

Comment:
@jpweber when should it be $labels.deployment and when should it be $labels.release?

Reply:
They should generally always be deployment, but some of the queries don't actually have deployment as part of their labels. So we use release there. They hold the same data so they are interchangeable as long as one of them isn't empty.